control_apps/src/control_mngr.rs

```diff
@@ -19,6 +19,7 @@ const CTRL_ALTITUDE_FLOOR: f32 = 15000.0; // minimum allowed altitude in meters
 const CTRL_ERROR_DEADZONE: f32 = 100.0; // magnitude of margin to allow without actuation
 const CTRL_ERROR_READY_THRESHOLD: f32 = 1000.0; // basically opposite of deadzone
 const CTRL_SPEED_DEADZONE: f32 = 0.2; // magnitude of margin to allow without actuation
+const CTRL_TELEM_MAX_AGE: Duration = Duration::from_secs(2); // maximum age of telemetry to act on
```

Comment:
[nit] How do you feel about standardizing on tlm? big ol nit dont feel the need to change

Reply:
You mean TLM instead of TELEM? sounds good to me